src/main/io/serial_4way.c

```diff
@@ -332,10 +332,8 @@ uint16_t _crc_xmodem_update (uint16_t crc, uint8_t data) {
 #define ATMEL_DEVICE_MATCH ((pDeviceInfo->words[0] == 0x9307) || (pDeviceInfo->words[0] == 0x930A) || \
         (pDeviceInfo->words[0] == 0x930F) || (pDeviceInfo->words[0] == 0x940B))
 
-#define SILABS_DEVICE_MATCH ((pDeviceInfo->words[0] == 0xF310) || (pDeviceInfo->words[0] == 0xF330) || \
-        (pDeviceInfo->words[0] == 0xF410) || (pDeviceInfo->words[0] == 0xF390) || \
-        (pDeviceInfo->words[0] == 0xF850) || (pDeviceInfo->words[0] == 0xE8B1) || \
-        (pDeviceInfo->words[0] == 0xE8B2))
+#define SILABS_DEVICE_MATCH (pDeviceInfo->words[0] > 0xE800) // && (pDeviceInfo->words[0] < 0xF900))
```

Comment:
Until now we were very specific about device info. Why remove the limit here - maybe extent the range:
```suggestion
#define SILABS_DEVICE_MATCH (pDeviceInfo->words[0] > 0xE800) && (pDeviceInfo->words[0] < 0xF900))
```

Reply:
@haslinghuis No big deal, changed now...